### PR TITLE
[stable/rabbitmq-ha] add option for StatefulSet annotations

### PR DESCRIPTION
**Pull request by max-rocket-internet**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by max-rocket-internet** [comment]
@lachie83 or @steven-sheehy can someone please review?

File: stable/rabbitmq-ha/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.36.4
+version: 1.36.5
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: 1.37.0 for new features

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Other than version bump it LGTM

File: stable/rabbitmq-ha/Chart.yaml (hunk 1 above)
Comment (max-rocket-internet) [reply stable/rabbitmq-ha/Chart.yaml]: OK done

**Comment by max-rocket-internet** [comment]
I've read the logs the test and can't really see what the problem is?

**Comment by steven-sheehy** [comment]
Think there's a general issue with the e2e test

**Review by steven-sheehy — APPROVED** [review]
/lgtm

**Pull request merged**